docs/cli/enterprise.md

```diff
@@ -0,0 +1,219 @@
+# Gemini CLI for the Enterprise
+
+This document outlines configuration patterns and best practices for deploying and managing Gemini CLI in an enterprise environment. By leveraging system-level settings, administrators can enforce security policies, manage tool access, and ensure a consistent experience for all users.
+
+> **A Note on Security:** The patterns described in this document are intended to help administrators create a more controlled and secure environment for using Gemini CLI. However, they should not be considered a foolproof security boundary. A determined user with sufficient privileges on their local machine may still be able to circumvent these configurations. These measures are designed to prevent accidental misuse and enforce corporate policy in a managed environment, not to defend against a malicious actor with local administrative rights.
+
+## Centralized Configuration: The System Settings File
+
+The most powerful tool for enterprise administration is the system-wide `settings.json` file. This file allows you to define a baseline configuration that applies to all users on a machine, overriding any user or project-level settings.
```

Comment:
are the settings merged with a precedence order? or completely overridden? let's make it clear here, especially given that we talk about merging/precedence for mcp servers below